<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>558</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="body" native="true">
     <property name="font">
      <font>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1" rowspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QWidget" name="group" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QFrame" name="frame1">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout1">
                 <item>
                  <widget class="QComboBox" name="comboBox1">
                   <property name="minimumSize">
                    <size>
                     <width>48</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                     <kerning>false</kerning>
                    </font>
                   </property>
                   <item>
                    <property name="text">
                     <string>p1</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>p2</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>p3</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>p4</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="equal1">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                     <kerning>false</kerning>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">margin-left: 10%; margin-right: 10%;</string>
                   </property>
                   <property name="text">
                    <string>=</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinBox1">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                     <kerning>false</kerning>
                    </font>
                   </property>
                   <property name="maximum">
                    <number>100</number>
                   </property>
                   <property name="value">
                    <number>100</number>
                   </property>
                   <property name="displayIntegerBase">
                    <number>10</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit1">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>200</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                     <kerning>false</kerning>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">margin-left: 10%; margin-right: 10%;</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame2">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout2">
                 <item>
                  <widget class="QComboBox" name="comboBox2">
                   <property name="minimumSize">
                    <size>
                     <width>48</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                     <kerning>true</kerning>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="currentIndex">
                    <number>1</number>
                   </property>
                   <item>
                    <property name="text">
                     <string>p1</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>p2</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>p3</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>p4</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="equal2">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                     <kerning>false</kerning>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">margin-left: 10%; margin-right: 10%;</string>
                   </property>
                   <property name="text">
                    <string>=</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinBox2">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                     <kerning>false</kerning>
                    </font>
                   </property>
                   <property name="maximum">
                    <number>100</number>
                   </property>
                   <property name="value">
                    <number>65</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit2">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>200</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                     <kerning>false</kerning>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">margin-left: 10%; margin-right: 10%;</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame3">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout3">
                 <item>
                  <widget class="QComboBox" name="comboBox3">
                   <property name="minimumSize">
                    <size>
                     <width>48</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                     <kerning>false</kerning>
                    </font>
                   </property>
                   <property name="currentIndex">
                    <number>2</number>
                   </property>
                   <item>
                    <property name="text">
                     <string>p1</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>p2</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>p3</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>p4</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="equal3">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                     <kerning>false</kerning>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">margin-left: 10%; margin-right: 10%;</string>
                   </property>
                   <property name="text">
                    <string>=</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinBox3">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                     <kerning>false</kerning>
                    </font>
                   </property>
                   <property name="maximum">
                    <number>100</number>
                   </property>
                   <property name="value">
                    <number>30</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit3">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>200</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                     <kerning>false</kerning>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">margin-left: 10%; margin-right: 10%;</string>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame4">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout4">
                 <item>
                  <widget class="QComboBox" name="comboBox4">
                   <property name="minimumSize">
                    <size>
                     <width>48</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                     <kerning>false</kerning>
                    </font>
                   </property>
                   <property name="currentIndex">
                    <number>3</number>
                   </property>
                   <item>
                    <property name="text">
                     <string>p1</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>p2</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>p3</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>p4</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="equal4">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                     <kerning>false</kerning>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">margin-left: 10%; margin-right: 10%;</string>
                   </property>
                   <property name="text">
                    <string>=</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinBox4">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                     <kerning>false</kerning>
                    </font>
                   </property>
                   <property name="maximum">
                    <number>100</number>
                   </property>
                   <property name="value">
                    <number>23</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit4">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>200</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                     <kerning>false</kerning>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">margin-left: 10%; margin-right: 10%;</string>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="controlBox">
        <property name="title">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QPushButton" name="okButton">
           <property name="text">
            <string>Ок</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetButton">
           <property name="text">
            <string>Сброс</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exitButton">
           <property name="text">
            <string>Выход</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
